client/src/components/composite/SignUpForm/Sections/PaymentSection.tsx

```diff
@@ -11,54 +11,76 @@ import {
 } from "services/AppData/AppDataQueries"
 import { ACCOUNT_SETUP_ROUTE } from "../utils/RouteNames"
 import { useMembershipPaymentDetails } from "store/MembershipPayment"
-
 type PaymentSectionProps = { wantsBankTransfer: (newState: boolean) => void }
 
 const BankTransferSection = ({ wantsBankTransfer }: PaymentSectionProps) => {
   const navigate = useNavigate()
   const [{ currentUser }] = useAppData()
   const { data: prices } = useMembershipPricesQuery()
-  const { data } = useBankPaymentDetailsQuery()
-
   const [{ membershipType }] = useMembershipPaymentDetails()
-
   const { data: userMembershipDetails } =
     useMembershipClientSecretQuery(membershipType)
-
-  /**
-   * Use data fetched to find the correct price
-   */
+  const { data } = useBankPaymentDetailsQuery()
   const requiredPrice = prices?.find(
     (price) => price.type === userMembershipDetails?.membershipType
   )
+  /**
+   * Use data fetched to find the correct price
+   */
+
+  const CopyButton = (text: any) => {
+    const handleOnclick = () => {
+      navigator.clipboard.writeText(text)
```

Comment:
and
```suggestion
  try {
    await navigator.clipboard.writeText(text);
  } catch (error) {
    console.error(error.message);
}
```
its asynchronous

Reply:
cheers bro